src/commonMain/kotlin/fr/acinq/lightning/io/Peer.kt

```diff
@@ -200,26 +202,38 @@ class Peer(
         }
         launch {
             finalWallet.walletStateFlow
-                .distinctUntilChangedBy { Pair(it.unconfirmedBalance, it.confirmedBalance) }
+                .distinctUntilChangedBy { it.totalBalance }
```

Comment:
This is not equivalent, the previous code displayed a log line when a tx was confirmed.

Reply:
True, but do we need that?